### PR TITLE
chore: Update 3rd party DEPENDENCIES file

### LINKED ISSUE #281: Update 3rd party DEPENDENCIES file
The `DEPENDENCIES` seems to be out of date. It does contain restricted dependencies like [here](https://github.com/eclipse-tractusx/tractusx-edc/blob/5eb0eb33b7c9340c62a7eeaa7bbb0b232cd8f834/DEPENDENCIES#L4) for example.

`restricted` dependencies are allowed during development, as long as the IP issues are created with dash. You would then have issue numbers in the last column of the `DEPENDENCIES` file. For a release, all the `restricted` and of course also `rejected` issues should be resolved.

Please run the [Dash licence tool](https://github.com/eclipse/dash-licenses) again to update the `DEPENDENCIES` file

### PR DESCRIPTION
**Pull request by tuncaytunc-zf**
## WHAT

Update 3rd party DEPENDENCIES file

## WHY

The DEPENDENCIES file was out of date

## FURTHER NOTES

Closes #281

### CODE REVIEW

**Comment by SebastianBezold** [comment]
Hi @tuncaytunc-zf, thanks for updating the dependencies. 
I can see some of the dependencies are marked as `restricted` this is can be the case, if there is no information about the library+version in the Eclipse Foundation IP database.
Before merging, we need to at least open the IP issues for the restricted libraries. 
This can be done via dash tool by any committer with the `-review` option. If you can provide me with the input file, that you where using for your dash run, I'm happy to create the issues for you

**Comment by tuncaytunc-zf** [comment]
> IP issues created:
> 
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7926
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7927
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7928
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7929
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7930
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7931
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7932
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7933
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7934
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7935
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7936
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7937
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7938
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7939
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7940
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7941
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7942
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7943
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7944
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7945
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7946
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7947
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7948
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7949
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/7950

Thank you for creating all this issues!
Once this are done, will update the DEPENDENCIES file again.

**Review by SebastianBezold — APPROVED** [review]

**Pull request merged**
